packages/app-desktop/gui/NoteEditor/NoteBody/TinyMCE/TinyMCE.tsx

```diff
@@ -1229,8 +1229,8 @@ const TinyMCE = (props: NoteBodyEditorProps, ref: any) => {
 
 		// eslint-disable-next-line @typescript-eslint/no-explicit-any -- Old code before rule was applied
 		async function onCut(event: any) {
-			const selectedContent = editor.selection.getContent();
-			copyHtmlToClipboard(selectedContent);
```

Comment:
so you removed the line where copying to clipboard took place, how is that even supposed to work?

Reply:
> so you removed the line where copying to clipboard took place, how is that even supposed to work?

When I copy paste an entire checklist I saw in the logs that the pasted HTML had a class name joplin-checklist, and when I saw the logs of cut event I didn't saw the class given to the checklist, so I removed  copyHtmlToClipboard(selectedContent); to see if there's any default/global cut event working and I saw something like this when I cut pasted after removing
 - [ ] []
- [ ] test
- [ ] test2

There was an extra square bracket added to the selected content, but I wasn't getting bullet markers instead on checkbox so, the cut was working fine, I solved this extra bracket problem in changes of resourceHandling.ts